packages/frontend/src/providers/ExplorerProvider.tsx

```diff
@@ -239,6 +239,7 @@ export interface ExplorerReduceState {
         customDimension: {
             isOpen: boolean;
             isEditing?: boolean;
+            table?: string;
```

Comment:
So we know to what table we need to add the custom SQL dimension. The Bin option gets that info from the selected dimension which is not available for SQL option.